src/keycloak/chart/templates/keycloak-logging-alerting-rules.yaml

```
@@ -0,0 +1,51 @@
+# Copyright 2025 Defense Unicorns
+# SPDX-License-Identifier: AGPL-3.0-or-later OR LicenseRef-Defense-Unicorns-Commercial
+
+{{- if .Values.detailedObservability }}
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: {{ include "keycloak.fullname" . }}-logging-alerts
+  namespace: {{ .Release.Namespace }}
+  labels:
+    prometheus: kube-prometheus-stack-prometheus
+    app.kubernetes.io/name: {{ include "keycloak.name" . }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+    app.kubernetes.io/component: keycloak
+spec:
+  groups:
+    - name: keycloak-logging-alerts
+      rules:
+        - alert: KeycloakRealmModificationsDetected
+          expr: uds_keycloak:realm_modifications > 0
```

```suggestion
          expr: increase(uds_keycloak_realm_modifications_total[5m]) > 0
```
based on feedback below to make this metric a monotonic counter. this essentially checks if there is an increase in total admin events over the last 5 mins